pvanalytics/quality/outliers.py

```diff
@@ -1 +1,62 @@
 """Functions for identifying and labeling outliers."""
+import pandas as pd
+from scipy import stats
+
+
+def tukey(data, k=1.5):
+    r"""Identify outliers based on the interquartile range.
+
+    A value is considered an outlier if it does *not* satisfy the
+    following condition
+
+    .. math::
+        Q_1 - k(Q_3 - Q_1) \le x \le Q_3 + k(Q_3 - Q_1)
+
+    where :math:`Q_1` is the value of the first quartile and
+    :math:`Q_3` is the value of the third quartile.
+
+    Parameters
+    ----------
+    data : Series
+        The data in which to find outliers.
+    k : float, default 1.5
+        Multiplier of the IQR. Using a larger value will be more
+        permissive of values that are far from the median.
+
+    Returns
+    -------
+    Series
+        A series of booleans with True for each value that is not an
```

Comment:
Should the function maintain the grammar, for readability? As written, `quality.outliers.tukey` returns `True` for outliers, which means that the code using the function would read

```
outlier = quality.outliers.tukey(x)
if not outlier:
   do stuff with outliers
```

Reply:
As written this returns `True` for any value that is _not_ an outlier. I wrote it in this way to make it easy to exclude outliers:

```python
no_outliers = data[outliers.tukey(data)]
```

Is your suggestion to return `False` for any value that is not an outlier and `True` for outliers?

```python
no_outliers = data[~outliers.tukey(data)]
```

Now that I write those examples out (along with your example above) the latter seems like a better choice.